toolkit/motors/rev_motors.py

```diff
@@ -79,6 +73,14 @@ def init(self):
         """
         Initializes the motor controller, pid controller, and encoder
         """
+        if self._init_complete:
+            print("SparkMax already initialized")
```

Comment:
Can we do better than "print". Could we start doing something more like logging....

Reply:
yeah thats my bad, it was mainly for debugging. when I make the updates Ill change it to our logging system

Reply:
There should be a way to make logging help with debugging so that our default is logging and staying away from printing. I want to make this the standard for all of our programmers.